### ISSUE TITLE
Can't get values at fixed points anymore

### ISSUE THREAD

**Issue by ianhbell** [usnistgov/REFPROP-issues#464]
**EDIT** This is broken for NBP, CRIT as well

I tried to do this in C++:
```C++
// At the triple point, get the densities
std::vector<double> z(20, 0.0);
auto rTRP = REFPROP("NITROGEN", "TRIP", "DLIQ;DVAP", MOLAR_BASE_SI, 0, 0, 0, 0, z);
```
and it failed with
```
[ALFLSH error 1] One or more inputs are out of range; Temperature below triple-point or minimum temperature: T = 0 K, Tmin = 63.1510 K.
```
and I confirmed I get the same bug in Python with the development version:
```python
import os
os.environ['RPPREFIX'] = r'C:\Users\ihb\Code\REFPROP-cmake\bld\Release'

import os
import sys
from ctREFPROP.ctREFPROP import REFPROPFunctionLibrary

RPPREFIX = os.getenv("RPPREFIX")
RP = REFPROPFunctionLibrary(RPPREFIX)
RP.SETPATHdll(RPPREFIX)

print("SYSTEM", sys.platform)
print("REFPROP", RP.RPVersion())
print("")

r = RP.REFPROPdll("NITROGEN", "TRIP", "DLIQ;DVAP", RP.MOLAR_BASE_SI, 0, 0, 0, 0, [1])
print(f"{r.Output[0]=}")
print(f"{r.ierr=}")
print(f"{r.herr=}")
```
yielding
```
SYSTEM win32
REFPROP 10.0.1.13

r.Output[0]=-9999990.0
r.ierr=1
r.herr='[ALFLSH error 1] One or more inputs are out of range; Temperature below triple-point or minimum temperature: T = 0 K, Tmin = 63.1510 K.'
```

**Comment by ianhbell** [comment]
This worked in 10.0.0:
```
SYSTEM win32
REFPROP 10.0.0.02

r.Output[0]=30957.306220703686
r.ierr=0
r.herr=''
```

**Comment by ianhbell** [comment]
Fixed in 10.0.1.14
